src/components/Catalog/catalog.css

```diff
@@ -0,0 +1,98 @@
+a {
+  text-decoration: none;
+}
+li {
+  list-style: none;
+}
+body {
+  font-family: "Montserrat", sans-serif;
+}
+/* Courses Catalog */
+.courses-catalog {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  gap: 30px;
+  padding: 20px 10%;
+}
+.catalog-content {
+  grid-column: span 12;
+  text-align: center;
+  /* margin-bottom: 20px; */
+}
+.catalog {
+  grid-column: span 3;
+}
+.course,
+.course-explore {
+  margin-bottom: 15px;
+  font-weight: 700;
+}
+.course-text {
+  font-weight: 500;
+  font-size: 14px;
+}
+.yellow-text {
+  color: rgb(238, 241, 38);
+}
+.catalog {
+  padding: 20px;
+  background: rgba(245, 245, 220, 0.779);
+  transition: 0.5s;
+}
+.catalog:hover {
+  transform: translate(-5px, -20px);
+}
```

Comment:
- [ ] Let's hold on about animation for now. Kindly remove it. We might need to use a package later to add animation in a very cool way.